takwimu/sql/junior_secondary_school_enrollment.sql

```diff
@@ -0,0 +1,206 @@
+--
+-- PostgreSQL database dump
+--
+
+-- Dumped from database version 10.5
+-- Dumped by pg_dump version 10.5
+
+SET statement_timeout = 0;
+SET lock_timeout = 0;
+SET client_encoding = 'UTF8';
+SET standard_conforming_strings = on;
+SELECT pg_catalog.set_config('search_path', '', false);
+SET check_function_bodies = false;
+SET client_min_messages = warning;
+
+ALTER TABLE IF EXISTS ONLY public.junior_secondary_school_enrollment DROP CONSTRAINT IF EXISTS pk_junior_secondary_school_enrollment;
+DROP TABLE IF EXISTS public.junior_secondary_school_enrollment;
+SET default_tablespace = '';
+
+SET default_with_oids = false;
+
+--
+-- Name: junior_secondary_school_enrollment; Type: TABLE; Schema: public; Owner: -
+--
+
+CREATE TABLE public.junior_secondary_school_enrollment (
+    geo_level character varying(15) NOT NULL,
+    geo_code character varying(10) NOT NULL,
+    geo_version character varying(100) DEFAULT ''::character varying NOT NULL,
+    sex character varying(128) NOT NULL,
+    junior_secondary_school_enrollment_year character varying(128) NOT NULL,
+    total integer
+);
+
+
+--
+-- Data for Name: junior_secondary_school_enrollment; Type: TABLE DATA; Schema: public; Owner: -
+--
+
+COPY public.junior_secondary_school_enrollment (geo_level, geo_code, geo_version, sex, junior_secondary_school_enrollment_year, total) FROM stdin;
+country	NG	2009	Female	2015	2920182
+country	NG	2009	Male	2015	3260109
+country	NG	2009	Female	2016	2786332
+country	NG	2009	Male	2016	3181810
+level1	NG_1_001	2009	Female	2015	35347
+level1	NG_1_001	2009	Male	2015	31755
+level1	NG_1_001	2009	Female	2016	19633
+level1	NG_1_001	2009	Male	2016	15709
+level1	NG_1_002	2009	Female	2015	46664
+level1	NG_1_002	2009	Male	2015	55127
+level1	NG_1_002	2009	Female	2016	47614
+level1	NG_1_002	2009	Male	2016	56286
+level1	NG_1_003	2009	Female	2015	190263
+level1	NG_1_003	2009	Male	2015	177488
+level1	NG_1_003	2009	Female	2016	92432
+level1	NG_1_003	2009	Male	2016	88500
+level1	NG_1_004	2009	Female	2015	68637
+level1	NG_1_004	2009	Male	2015	61269
+level1	NG_1_004	2009	Female	2016	48548
+level1	NG_1_004	2009	Male	2016	46064
+level1	NG_1_005	2009	Female	2015	70142
+level1	NG_1_005	2009	Male	2015	99089
+level1	NG_1_005	2009	Female	2016	82239
+level1	NG_1_005	2009	Male	2016	112110
+level1	NG_1_006	2009	Female	2015	20674
+level1	NG_1_006	2009	Male	2015	21514
+level1	NG_1_006	2009	Female	2016	17472
+level1	NG_1_006	2009	Male	2016	17879
+level1	NG_1_007	2009	Female	2015	183168
+level1	NG_1_007	2009	Male	2015	234728
+level1	NG_1_007	2009	Female	2016	71648
+level1	NG_1_007	2009	Male	2016	83618
+level1	NG_1_008	2009	Female	2015	NA
+level1	NG_1_008	2009	Male	2015	NA
```

Comment:
These are supposed to be numbers, no?

Reply:
@kilemensi we shouldn't have null entry for total, I will remove the fields completely